aquadoggo/src/db/stores/document.rs

```diff
@@ -438,7 +441,10 @@ mod tests {
             .await
             .unwrap();
 
-        assert!(view_does_not_exist.is_none())
+        assert!(view_does_not_exist.is_none());
+
+        // Disconnect from database
+        db.close().await;
```

Comment:
We need to add this to every test, to make sure the connection is dropped before we can move over to the next test.